### PR TITLE
Add ability to expand/collapse all

### PR DESCRIPTION
**Pull request by chbonser**

### CODE REVIEW

File: addon/components/accordion-list.js [hunk 1]
```diff
@@ -13,11 +13,26 @@ const AccordionListComponent = Ember.Component.extend({
...
+  _registeredItems: null,
   init() {
     this._super(...arguments);
     this.set('_activeItems', Ember.A([]));
+    this.set('_registeredItems', Ember.A([]));
   },
 
+  // If each registered item has at least on panel open, then return true
+  allExpanded: computed('_activeItems.[]', '_registeredItems.[]', function() {
+    let { _activeItems, _registeredItems } = this.getProperties('_activeItems', '_registeredItems');
```
Comment (rhlowe) [inline addon/components/accordion-list.js]: Destructuring, mmmmmmmm.

File: addon/components/accordion-list.js [hunk 2]
```diff
@@ -13,11 +13,26 @@ const AccordionListComponent = Ember.Component.extend({
...
+
+    for (let i=0; i < _registeredItems.length; i++) {
```
Comment (rhlowe) [inline addon/components/accordion-list.js]: An `array.reduce` might be cleaner here.

Comment (chbonser) [reply addon/components/accordion-list.js]: good call. thanks.

File: addon/components/accordion-list.js [hunk 3]
```diff
@@ -13,11 +13,21 @@ const AccordionListComponent = Ember.Component.extend({
...
     this._super(...arguments);
     this.set('_activeItems', Ember.A([]));
+    this.set('_registeredItems', Ember.A([]));
   },
 
+  // If each registered item has at least on panel open, then return true
+  allExpanded: computed('_activeItems.[]', '_registeredItems.[]', function() {
+    let { _activeItems, _registeredItems } = this.getProperties('_activeItems', '_registeredItems');
+
+    let anyMissing = _registeredItems.reduce((anyMissing, item) => anyMissing || isNone(_activeItems.findBy('id', item.itemId)), false);
```
Comment (boydkr) [inline addon/components/accordion-list.js]: `Array.some()` will short-circuit completion, and you don't have to deal with the accumulator

**Comment by chbonser** [comment]
merging and will follow up with a PR to upgrade dependencies and fix the test suite

**Pull request merged**